src/IssueViz/Editor/IIssueSpanCalculator.cs

```diff
@@ -32,7 +32,7 @@ public interface IIssueSpanCalculator
         /// Returns the text span corresponding to the supplied analysis issue location.
         /// Returns null if the location line hash is different from the snapshot line hash
         /// </summary>
-        SnapshotSpan CalculateSpan(IAnalysisIssueLocation location, ITextSnapshot currentSnapshot);
+        SnapshotSpan CalculateSpan(ITextRange range, ITextSnapshot currentSnapshot);
```

Comment:
The calculator was modified to receive textRange now, instead of the location, so it would work for quick fixes too